pipit/trace.py

```diff
@@ -16,6 +16,15 @@ def __init__(self, definitions, events):
         self.definitions = definitions
         self.events = events
 
+        # list of numeric columns which we can calculate inc/exc metrics with
+        self.numeric_cols = list(
```

Comment:
list of numeric columns with which we can do inc/exc calculations
process and thread aren't in this as they are categorical